### PR TITLE
orientdb: update to 3.1.11

### PR DESCRIPTION
**Pull request by digitalagedragon**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

Updated to version 3.1.11 and template / update files fixed to point to the new upstream host per #30041.

### CODE REVIEW

File: srcpkgs/orientdb/template [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 conf_files="/etc/${pkgname}/*"
-wrksrc="${pkgname}-community-${version}"
 depends="virtual?java-runtime"
 short_desc="Distributed Graph Database"
 maintainer="bougyman <bougyman@rubyists.com>"
 license="Apache-2.0"
+homepage="https://orientdb.org"
+distfiles="https://s3.us-east-2.amazonaws.com/orientdb3/releases/${version}/orientdb-${version}.tar.gz"
+checksum=73830279a5587de17ea4f5864598a0829bb03212d6cd36d185fe126a46086236
 patch_args="-p1"
```
Comment (ericonr) [inline srcpkgs/orientdb/template:14]: When you're moving this (see xlint test result), please also use `-Np1`

Comment (ericonr) [reply srcpkgs/orientdb/template:14]: Well, now my comment is outdated :p

Comment (digitalagedragon) [reply srcpkgs/orientdb/template:14]: Ugh, saw this 30 seconds too late...

**Comment by digitalagedragon** [comment]
wow I am being very good at git today

**Comment by ericonr** [comment]
Well, this looks great, thanks!

Will let it sit for a bit so @bougyman can leave thoughts.

**Pull request merged**